build-artifacts/project-template-gradle/build.gradle

```diff
@@ -721,6 +721,26 @@ copyTypings.onlyIf({
 	generateTypescriptDefinitions.didWork;
 })
 
+task validateAppIdMatch {
+	doLast {
```

Comment:
Please check if the build is happening in debug or release - https://github.com/NativeScript/android-runtime/blob/b74218bfd5d0e57087dcc0473366f76d34ff6433/build-artifacts/project-template-gradle/build.gradle#L84

There is no point in the user seeing the warning if they are preparing to release their app with an applicationId they themselves put in the app.gradle.